libs/libvirtio/plat_include/zynqmp/virtio/virtio_plat.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2019, DornerWorks
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#define IRQ_SPI_OFFSET 32
```

Comment:
Come comments here would be nice to explain where this values come from.

Reply:
This is standard for ARMv8 and is in the other platform specific virtio_plat.h headers.

@yyshen might have more clarification, but my understanding is that each processor has 32 Private Peripheral Interrupts (PPI). SPI (Shared Peripheral Interrupts) start at 32, hence the offset.

Reply:
I couldn't mark all three lines. The `IRQ_SPI_OFFSET` is somehow clear (could still move to a shared header eventually), but the `61` that is used twice could get a comment. Is this a random pick or is there a spec?

Reply:
Honestly I have no idea where that came from. If you do the `61 + 32` that's actually a GEM2 interrupt, which is an unused hardware device. The value should probably be updated to not interfere with a physical device. I'll have to test.

Reply:
I tested `158`, which doesn't correspond to a physical interrupt. I'll add a comment explaining the value

Reply:
yes, the first 32 IRQs are reserved for PPI and SGI. The IRQs for peripherals start from 32.

Reply:
Cab you add a comment in the code why the same interrupt us used for both channels? Is this driven by implementation needs?

Reply:
That is the case for each platform with VPCI support. Looking at the vpci generation code, that doesn't seem to be a requirement. So I thought it would be best to just match the behavior of previously supported platforms. We can track an issue to make each virtio device have its own interrupt.